lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -127,6 +135,82 @@ def file_updated?(change, path)
 
         false
       end
+
+      sig { returns(T.nilable(T::Boolean)) }
+      def git_repo?
+        _, status = Open3.capture2e("git rev-parse --is-inside-work-tree")
+
+        status.success?
+      end
+
+      sig { returns(T::Boolean) }
+      def lockfile_changed?
+        !fetch_lockfile_diff.empty?
```

Comment:
```suggestion
        fetch_lockfile_diff
        !@lockfile_diff.empty?
```
(to avoid violating https://en.wikipedia.org/wiki/Command%E2%80%93query_separation )